upgrade-2.x.sh

```diff
@@ -170,6 +170,16 @@ function remove_containers() {
     ${DOCKER_CMD} rm $(${DOCKER_CMD} ps -a -q) > /dev/null 2>&1 || true
 }
 
+function remove_rec_files() {
+    echo "Removing REC files to make space for the update."
+    local boot_dir='/mnt/boot'
+    for f in ${boot_dir}/*.REC; do
+        log "Removing $f from boot partition"
+        rm -f $f
+    done
```

Comment:
The code above will print a message even if there are no REC files:
For example, 
```
tmpdir=$(mktemp -d)
for f in ${tmpdir}/*.REC; do echo $f; done
/tmp/tmp.RUcM4kNJiB/*.REC
```

We  want to add a warning message only if REC files are present as they should not be, so maybe something like the following works better:

```
find "${boot_dir}" -name "*.REC" -exec log WARN "Removing {} from boot partition" \; -exec  rm -f {} \;
```

Reply:
`-exec log WARN "Removing {} from boot partition" \;` has trouble reading the file. Tested a few more variants before settling on using `shopt -s nullglob` with the original loop. Let me know if there are better options